Source/WebCore/page/DOMTimer.cpp

```diff
@@ -425,30 +417,6 @@ std::optional<MonotonicTime> DOMTimer::alignedFireTime(MonotonicTime fireTime) c
     return adjustedFireTime - (adjustedFireTime % alignmentInterval) + alignmentInterval + randomizedOffset;
 }
 
-void DOMTimer::makeOpportunisticTaskDeferralScopeIfPossible(ScriptExecutionContext& context)
-{
-    if (!m_oneShot || m_currentTimerInterval <= 1_ms) {
-        // FIXME: This should eventually account for repeating timers and one-shot timers that were
-        // previously scheduled, and are about to fire.
-        return;
-    }
-
-    RefPtr document = dynamicDowncast<Document>(context);
-    if (!document)
-        return;
-
-    auto* page = document->page();
-    if (!page)
-        return;
-
-    m_opportunisticTaskDeferralScope = page->opportunisticTaskScheduler().makeDeferralScope();
-}
-
-void DOMTimer::clearOpportunisticTaskDeferralScopeIfPossible()
```

Comment:
This is still declared in DOMTimer.h but should be removed.

Reply:
Removed.